vendor/github.com/giantswarm/certificatetpr/custom_object.go

```diff
@@ -34,12 +36,19 @@ func (c *CustomObject) GetObjectMeta() meta.Object {
 type CustomObjectCopy CustomObject
 
 func (c *CustomObject) UnmarshalJSON(data []byte) error {
+	fmt.Fprintf(os.Stdout, "IN CustomObject UnmarshalJSON data: %s \n", data)
```

Comment:
This is the "decoding magic" from the k8s.io/client-go example. Plus the debug I hacked in.

See  https://github.com/kubernetes/client-go/blob/v2.0.0/examples/third-party-resources/types.go#L66